mne/tests/test_epochs.py

```diff
@@ -3181,7 +3282,9 @@ def test_drop_epochs():
     # Bound checks
     pytest.raises(IndexError, epochs.drop, [len(epochs.events)])
     pytest.raises(IndexError, epochs.drop, [-len(epochs.events) - 1])
-    pytest.raises(ValueError, epochs.drop, [[1, 2], [3, 4]])
+    pytest.raises(TypeError, epochs.drop, [[1, 2], [3, 4]])
+    with pytest.raises(TypeError):
```

Comment:
should have `match="..."` to both make it clear what's being caught, and make sure that's what is actually being caught